diff -ur mpc-0.8.1-original/Makefile.am mpc-0.8.1-patched/Makefile.am
--- mpc-0.8.1-original/Makefile.am	2008-12-03 12:35:48.000000000 +0300
+++ mpc-0.8.1-patched/Makefile.am	2011-04-28 17:18:02.000000000 +0400
@@ -1,5 +1,5 @@
 ## Makefile.am -- Process this file with automake to produce Makefile.in
-SUBDIRS = src tests doc
+SUBDIRS = src tests
 
 include_HEADERS = src/mpc.h
 EXTRA_DIST = Makefile.vc tests/tgeneric.c
diff -ur mpc-0.8.1-original/Makefile.in mpc-0.8.1-patched/Makefile.in
--- mpc-0.8.1-original/Makefile.in	2009-12-06 20:04:06.000000000 +0300
+++ mpc-0.8.1-patched/Makefile.in	2011-04-28 17:18:16.000000000 +0400
@@ -240,7 +240,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = src tests doc
+SUBDIRS = src tests
 include_HEADERS = src/mpc.h
 EXTRA_DIST = Makefile.vc tests/tgeneric.c
 all: config.h
